turbopack/crates/turbopack-ecmascript-runtime/js/src/shared/runtime-types.d.ts

```diff
@@ -13,6 +13,7 @@ type ChunkListScript = CurrentScript & { readonly brand: unique symbol }
 type ChunkPath = string & { readonly brand: unique symbol }
 type ChunkScript = CurrentScript & { readonly brand: unique symbol }
 type ChunkUrl = string & { readonly brand: unique symbol }
+// TODO this should actually be `string | number`
```

Comment:
!

At google I referred to this as 'The fundamental rule of JavaScript: Anywhere you expect a number, someone will pass you a string and visa versa'

Reply:
-> https://github.com/vercel/next.js/pull/80605